libraries/x86/test/test_x86_cmd.c

```diff
@@ -4,7 +4,7 @@
 #include "unity.h"
 #include "x86_cmd.h"
 
-static char s_cmd[30];
+static const char s_cmd[30];
```

Comment:
?

Please make sure to clean up your branch and remove any changes that are unrelated to your PR.